src/target/linux.rs

```diff
@@ -24,17 +24,17 @@ pub use writemem::WriteMemory;
 
 lazy_static::lazy_static! {
     pub static ref PAGE_SIZE: usize = unsafe { libc::sysconf(libc::_SC_PAGESIZE) as usize };
-    #[cfg(target_arch="x86_64")]
+    #[cfg(feature="x86_64")]
```

Comment:
Ptrace can only work with the local arch, so using `target_arch` makes more sense to me. Also enabling the x86_64 feature on other archs won't compile, as libc won't jave a u_debugreg field for the `libc::user` struct.